webpack/components/AnsibleHostDetail/components/RolesTab/index.js

```diff
@@ -62,6 +75,14 @@ const RolesTab = ({ hostId, history, canEditHost }) => {
           canEditHost={canEditHost}
         />
       )}
+      <Route path="/Ansible/roles/all">
```

Comment:
it was necessary to add it here as well, so the `AllRolesModal` component would appear.